src/mods/friendsList/index.js

```diff
@@ -16,10 +13,10 @@ function customFriendsList() {
 	$elixirIcon.parentNode.insertBefore(friendsIconElement, $elixirIcon.nextSibling);
 
 	// Create the friends list UI
-	document.querySelector('.js-friends-list-icon').addEventListener('click', createFriendsList);
+	document.querySelector('.js-friends-list-icon').addEventListener('click', toggleFriendsList);
```

Comment:
Nice change! Having all the logic in the `ui` and a `windowState` makes this so much easier than it originally would've been. Cool to see.